modules/rosa-hcp-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-aws-amis.adoc
+
+[id="rosa-hcp-aws-rhcos-ami_{context}"]
+= ROSA AMIs for {hcp-title}
+
+Red Hat provides RHCOS AMIs that are valid for the supported AWS regions and instance types that you can manually specify for your {hcp-title} nodes.
+
+.{hcp-title-first} AMIs for 4.15
```

Comment:
Is there a reason only 4.15 is included in the list?

Reply:
Our docs are built off of GA +1. Should we cover ALL ROSA versions, or just GA +1?

Reply:
We should cover all ROSA versions that are currently supported (https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.html#sd-life-cycle-dates_rosa-life-cycle).

Reply:
I've asked to clarify the lifecycle of AMIs for the data plane worker nodes in the ROSA HCP slack channel. I'm not sure if they follow the same lifecycle where the version you install the cluster on is the same AMI it will use in perpetuity or if the AMI will change as you upgrade. Once we get confirmation on that I think we can build the list.

With that said - I think since this only applies to HCP AMIs we should only go back as far as 4.14 as that was the version where HCP went GA.